include/swift/Remote/MetadataReader.h

```diff
@@ -261,7 +263,130 @@ class MetadataReader {
 
     return start;
   }
-  
+
+  /// Given a pointer to the metadata, attempt to read the value
+  /// witness table.
```

Comment:
You should probably mention here in this comment that it's not safe to access any non-mandatory members of the VWT, like extra inhabitants or enum members.

Reply:
Sure.